tests/pipeline/test_node.py

```diff
@@ -300,7 +300,7 @@ def dummy_func_args(**kwargs):
     return dummy_func_args, "A", "B"
 
 
-lambda_identity = lambda input1: input1  # noqa: disable=E731
+lambda_identity = lambda input1: input1  # noqa: disable=E731 # pylint: disable=C3001
```

Comment:
FYI I intentionally used the error code so as to circumvent any line length issues. I think that's fair, since this is a test.